sklearn/ensemble/_gb.py

```diff
@@ -1451,6 +1451,14 @@ class GradientBoostingClassifier(ClassifierMixin, BaseGradientBoosting):
     ...     max_depth=1, random_state=0).fit(X_train, y_train)
     >>> clf.score(X_test, y_test)
     0.913...
+
+    See :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_oob.py` for
+    an example on using Out-of-Bag estimates to estimate the optimal number of
+    iterations for Gradient Boosting.
```

Comment:
this example is linked from the user guide, and needs improvements, then we can think of adding it in the docstring, so removing it from here